src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/CommandResult.java

```diff
@@ -17,13 +17,16 @@
     /** The application should exit. */
     private final boolean exit;
 
+    private final int travelPlanObjectType;
```

Comment:
Maybe can add a Javadoc comment here. "Takes the value -1 if no tab changes are required. Otherwise, represents the index of the tab to set the travelPlanObjectListPanel to."

Reply:
Resolved.